### DISCUSSION TITLE
LaserGrbl positioning problem

### DISCUSSION THREAD

**Discussion by TT2DENKER** [arkypita/LaserGRBL#2363] [Q&A]
I think there is a new dimensioning in Lasergrbl versions. I am sending you the images below, now the starting point of the graphic to be engraved or cut is positioned 5-10 cm right and up from the zero point. For example, I use TwoTrees TS2, my cutting area is 45cm X 45 cm, but when I try to load a 40 cm engraving, the program says that these dimensions are high and does not position according to my working area. Is there anyone who has knowledge on this subject and can help?
![lasergrbl 1](https://github.com/user-attachments/assets/b0fb57de-23e3-409c-97a8-636004590baa)
![lasergrbl 2](https://github.com/user-attachments/assets/7394fcdb-efdc-4ef3-9314-02a300308501)

**Comment by Tribo-2dot0** [comment]
Digite $$ no console e cole o resultado aqui.

**Comment by TT2DENKER** [comment]
Where is the console section you are talking about? Could you please
explain in more detail? If possible, I would appreciate it if you could
describe it with screenshots.

Tribo-2dot0 ***@***.***>, 19 Eyl 2024 Per, 21:02 tarihinde
şunu yazdı:

**Comment by TT2DENKER** [comment]
Hello,
I wrote before, but I may not have explained it correctly. Therefore, I did
not receive a helpful answer. I have a TwoTrees TS2 10w laser machine. And
I do my engraving-cutting work with LaserGRBL program on this machine.
Recently, either I changed some settings without knowing it, or I started
to experience a problem after the new update files. As I mentioned above,
the working area of my TwoTrees TS2 10w machine is 450mm X 450mm. But as I
stated, after the new update, I can no longer cut or engrave 450X450mm in
the working area. I can't even work with 350X350mm dimensions. I
immediately get $130 and $131 warnings. Because the point where any file I
upload to the machine starts cutting/engraving is 10 cm to the right and
above from the (zero) point. No matter what I did, I could not move the
file to be processed on the LaserBRBL screen to the zero point. I am also
sending you the images in the attachment. If the file/image can be placed
at zero point on the program page, I will be able to cut or engrave in the
appropriate working area of the machine. My request from you is to help me
on how to correct the error in the images. Thank you in advance.


MARANGOZ WOODART ***@***.***>, 20 Eyl 2024 Cum, 09:47 tarihinde şunu
yazdı:

**Reply by Tribo-2dot0** [reply]
Se a imagem é um jpg/png pode haver área em branco ao redor dela, para isso use a ferramenta Auto Trim Image em LaserGrbl...
Se a imagem for um svg, abra-a em Inkscape, selecione o desenho e menu Editar > Ajustar Página à Seleção, salve...
O console está logo abaixo de Progress... digite $$ nele e cole o resultado aqui.
![Captura de tela 2024-08-08 084348](https://github.com/user-attachments/assets/781ac5d1-daf8-4ac3-a42d-5a9fff6cdcdf)

**Comment by TT2DENKER** [comment]
Dear friend, thank you very much. After I wrote $$ to the console, the
dimensioning was corrected, now it fits to zero point. But as I showed with
the marked places in the images below, you will see that even though my
laser machine's working area is 450X450mm, I continue to receive $130 and
$131 warnings for a 350mm file... I don't think it's very important, but I
would be glad if I could fix this error warning. Thank you again...

Tribo-2dot0 ***@***.***>, 22 Eyl 2024 Paz, 04:16 tarihinde
şunu yazdı:

**Comment by TT2DENKER** [comment]
Dear friend, I said in the previous message that the reset point in the
software was fixed, but I still have the same problem. I am sending you the
image of the parameters in the console after $$ in the attachment. I don't
understand why this problem occurs? It got fixed after I messaged you, I
thought it got fixed after the $$ sign, but it wasn't. Please help..

Tribo-2dot0 ***@***.***>, 22 Eyl 2024 Paz, 04:16 tarihinde
şunu yazdı:

**Reply by Tribo-2dot0** [reply]
Não vejo imagem nenhuma em anexo... pode copiar e colar o resultado de $$ aqui, como texto mesmo.

**Comment by TT2DENKER** [comment]
Tribo-2dot0 ***@***.***>, 19 Eyl 2024 Per, 21:02 tarihinde
şunu yazdı:
